gen/rules/src/main/scala/clue/gen/Generator.scala

```diff
@@ -236,7 +236,7 @@ trait Generator {
           val lensesDef = params.map { param =>
             val thisType  = qualifiedNestedType(nestPath, Type.Name(camelName))
             val childType = param.typeTree(nextPath, nextTypes)
-            q"implicit val ${Pat.Var(Term.Name(param.name))}:  monocle.Lens[$thisType, $childType] = monocle.macros.GenLens[$thisType](_.${Term
+            q"val ${Pat.Var(Term.Name(param.name))}:  monocle.Lens[$thisType, $childType] = monocle.macros.GenLens[$thisType](_.${Term
```

Comment:
Would it help on js size to declar them `lazy‘?

Reply:
Probably not after https://github.com/scala-js/scala-js/pull/4681? I don't know.

Reply:
Oh that's nice